bundles/SyliusReportBundle/index.rst

```diff
@@ -0,0 +1,15 @@
+SyliusReportBundle
+===================
+
+SyliusReportBundle is the flexible reports management system for Symfony2 e-commerce applications.
+
+Using this bundle, you can generate various and complex reports for your webshop, starting from defaults - user registrations report, and total sales report. Implemented renderers allows you to display data in many different ways - from simple but clear table, to impressive and colorful charts.
```

Comment:
for your webshop => for your **system**

starting from defaults - user registrations report, and total sales report => starting from user registrations or total sales report (<link do data fetcherow>)